client/app/components/faq/faqdetails.js

```diff
@@ -1,41 +1,27 @@
-import React, { Component } from 'react';
+import React from 'react';
```

Comment:
I prefer declaring components with `React.Component` instead of `Component`. So for example

```js

// bad
class Foo extends Component

// good
class Foo extends React.Component
```

This is purely a style/nitpick thing, open to other opinions.

Reply:
I use Component just because it looks neater, but I'm open to both.